webapp/app/components/header/AppLinks.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { NavLink, useLocation } from 'react-router-dom';
+
+const AppLinks = ({ i18n }) => {
+  const { pathname } = useLocation();
+  const isStatisticalFactsheets = pathname.includes("statisticalFactsheets");
+  return <NavLink
+    activeClassName="hidden"
+    to={`/${isStatisticalFactsheets ? '' : 'statisticalFactsheets'}`} className="app-header__menu-item">
+    {isStatisticalFactsheets ? "FRA Platform" : i18n.t('common.statisticalFactsheets')}
+  </NavLink>;
+};
```

Comment:
- Add TWO NavLinks one for Fra Platform and one for StatisticalFactsheets
- Link to Fra Platform 
- REMOVE ```;```